### DISCUSSION TITLE
Last open Panel crashes CTRL - how to delete it?

### DISCUSSION THREAD

**Discussion by antoniograzioli** [RomanKubiak/ctrlr#644] [General]
I am on a Mac, if I try to open CTRL I have several errors in the last opened panel, which (I don't know why) disable the menu in CTRL (File, MIDI and so on...)
I have tried DELETING the last open panel, it is loaded again (lloks like it's "cached" somehow)

How do I launch CTRL and have him IGNORE last saved/used panel?

**Comment by damiensellier** [comment]
Delete the following file and you'll be good to go : 

WINDOWS : C:\Users\_username_\AppData\RoamingCtrlr\Ctrlr.settings
MAC : Users/your_username/Library/Preferences/Ctrlr/Ctrlr.settings
LINUX : ~/Ctrlr/Ctrlr.settings

**Reply by antoniograzioli** [reply]
Thanks a lot! 
Stupid me, I have thought about that, had a look around and forgot this is an hidden Library folder...
Thanks again, it's working now

**Reply by damiensellier** [reply]
cool! happy you found your way out the loop.

**Reply by dobo365** [reply]
I'm also always keeping an empty Ctrlr.settings file in case of even if I think it is created automatically

**Comment by unityconstruct** [comment]
For linux users seeing this later: the Ctrlr path is [ ~/Ctrlr ]
